build.sh

```diff
@@ -2,7 +2,7 @@
 # Build script
 set -eo pipefail
 build_tag=$1
-name=content-service
+name=knowlgbbportal
```

Comment:
knowlg-portal

Reply:
This is the job name and. it should be like this only